C/tests/c4QueryTest.cc

```diff
@@ -415,15 +415,8 @@ N_WAY_TEST_CASE_METHOD(C4QueryTest, "C4Query dict literal", "[Query][C]") {
 N_WAY_TEST_CASE_METHOD(C4QueryTest, "C4Query N1QL parse error", "[Query][C][N1QL][!throws]") {
     int errPos;
     C4Error error;
-    {
-        ExpectingExceptions x;
-        CHECK(c4query_new2(db, kC4N1QLQuery, "SELECT foo bar"_sl, &errPos, &error) == nullptr);
-    }
-    CHECK(errPos == 11);
-    CHECK(error.domain == LiteCoreDomain);
-    CHECK(error.code == kC4ErrorInvalidQuery);
 
-    C4Query *q = c4query_new2(db, kC4N1QLQuery, "SELECT foo, bar"_sl, &errPos, &error);
```

Comment:
Did you remove this test on purpose?

Reply:
I did not remove it, but extended it to include the new rule allowing adding alias without AS. This case was in the preceding case that expect it to fail.

Reply:
Where did this case move to?  I see two test cases before, and only one after.  

SELECT foo bar (aka SELECT foo AS bar) -> SELECT foo asAlias
SELECT foo, bar (SELECT FOO (and), BAR) -> ???

Reply:
Basically, I modified the second test case to include the logic of the first test case. Do you prefer not to touch the existing case, but modify the first one to, instead of expecting failure, expect it to succeed? I can do that.

Reply:
It seems this test has become redundant really.  The same logic is tested in another test, and the purpose of this test is no longer to check parsing errors.